Klaviyo classic destination migration info (#6668)
segmentio/segment-docs: src/connections/destinations/catalog/klaviyo/index.md

id: 54521fd825e721e32a72eec8
---

> warning "Segment will deprecate the Klaviyo Classic destination on June 30th, 2024"
> [Klaviyo will deprecate the endpoints used by this destination on June 2024](https://developers.klaviyo.com/en/docs/migrating_from_v1v2_to_the_new_klaviyo_apis){:target="_blank”}. Segment will not update this destination with the new endpoint, but will deprecate the destination. Users who want to send data to Klaviyo should migrate to the Klaviyo (Actions) destination. 
>

> > On June 7, 2024, Segment will migrate all Klaviyo classic destinations to the new Klaviyo (Actions) destination. For more information about this migration, see the [Migrate to the Klaviyo (Actions) destination](#migrate-to-the-klaviyo-actions-destination) documentation. 

Author: @rvadera12 could we say "Starting on June 7, 2024..."? It lets people know what to expect

> > On June 7, 2024, Segment will migrate all Klaviyo classic destinations to the new Klaviyo (Actions) destination. For more information about this migration, see the [Migrate to the Klaviyo (Actions) destination](#migrate-to-the-klaviyo-actions-destination) documentation. 

Author: ```suggestion
> Starting on June 7, 2024, Segment will migrate all Klaviyo classic destinations to the new Klaviyo (Actions) destination. **If you don't have an API key in the destination settings for your classic Klaviyo destination, you will need to take action.**
> 
> For more information about this migration, see the [Migrate to the Klaviyo (Actions) destination](#migrate-to-the-klaviyo-actions-destination) documentation. 
```

> info "Klaviyo deprecating v1/v2 APIs"
> Klaviyo will deprecate the endpoints used by this destination in June 2024. Segment will not update this destination with the new endpoint. Instead, Segment recommends customers switch to the new [Klaviyo Actions destination](/docs/connections/destinations/catalog/actions-klaviyo/), which already uses the new endpoints.

> ## Migrate to the Klaviyo (Actions) destination

Reviewer: Add a note that all new instances will have Migrated appended and here are some screenshots: 

<img width="1505" alt="Screenshot 2024-06-05 at 5 03 15 PM" src="https://github.com/segmentio/segment-docs/assets/89420099/5770fff7-a4d0-411f-ba88-f4fd5717865d">
<img width="937" alt="Screenshot 2024-06-05 at 5 08 00 PM" src="https://github.com/segmentio/segment-docs/assets/89420099/b78f0a4c-ef8b-4d52-b94c-905d3c6d7565">
<img width="1541" alt="Screenshot 2024-06-05 at 5 10 12 PM" src="https://github.com/segmentio/segment-docs/assets/89420099/b71cf406-ea70-428c-b349-aa32eab9bc5c">
<img width="930" alt="Screenshot 2024-06-05 at 5 14 01 PM" src="https://github.com/segmentio/segment-docs/assets/89420099/95478b06-638e-441a-9023-55a158b01eac">

Author: hi @rvadera12 - we avoid using screenshots in the docs - they're quickly outdated, add to site load time, and are not accessible. I'll add the note about the naming convention in the text, but I'd like to avoid adding any screenshots!

> On June 7th, 2024, Segment will automatically migrate all classic Klaviyo destinations to the new Klaviyo (Actions) destination. 

Author: ```suggestion
Starting on June 7th, 2024, Segment will automatically migrate all classic Klaviyo destinations to the new Klaviyo (Actions) destination. 
```

> info "Klaviyo deprecating v1/v2 APIs"

> > Klaviyo will deprecate the endpoints used by this destination in June 2024. Segment will not update this destination with the new endpoint. Instead, Segment recommends customers switch to the new [Klaviyo Actions destination](/docs/connections/destinations/catalog/actions-klaviyo/), which already uses the new endpoints.

Author: ```suggestion
> Klaviyo will deprecate the endpoints used by this destination on June 30, 2024. Segment will not update this destination with the new endpoint. Instead, Segment recommends customers switch to the new [Klaviyo Actions destination](/docs/connections/destinations/catalog/actions-klaviyo/), which already uses the new endpoints. For more information about the migration process, see the [Migrate to the Klaviyo (Actions) destination](#migrate-to-the-klaviyo-actions-destination) documentation.
```
